stl/inc/algorithm

```diff
@@ -386,6 +386,20 @@ namespace ranges {
     inline constexpr _For_each_n_fn for_each_n{_Not_quite_object::_Construct_tag{}};
 
     // VARIABLE ranges::find
+    // clang-format off
+    template <input_iterator _It, sentinel_for<_It> _Se, class _Ty, class _Pj = identity>
+        requires indirect_binary_predicate<ranges::equal_to, projected<_It, _Pj>, const _Ty*>

```

Comment:
Since these helpers started moving out of class scope, we've been applying a "familial" rule. A helper that's used only by algorithms within a single family like `_Is_heap_until_unchecked` doesn't have constraints since it's internal to a family. A helper that's used cross-family like `_Find_unchecked` has constraints for more strict enforcement since it's much more likely that not all callers are maintained as a unit with the helper.

Maybe we should make the distinction clearer by giving helpers with scopes larger than a single algorithm family their own `// SHOUTY BANNERS`?

Reply:
I would not have noticed that distinction because we are inconsistent about what gets `SHOUTY BANNERS`. A comment saying what you just said, `// used by several algorithms; concept-constrained for strict enforcement` or something would be better.

Reply:
Done